app/models/winning_bidder_export.rb

```diff
@@ -29,7 +34,19 @@ def header_values
       '13PP Vendor Zip Code',
       '13QQ Vendor Country Code',
       '13RR Vendor Phone Number',
-      '13SS Vendor Fax Number'
+      '13SS Vendor Fax Number',
+      '2B Effective Date',
+      '2C Current Completion Date',
+      '3A Base And All Options Value',
+      '3B Base And Exercised Options Value',
+      '3C Action Obligation',
+      '6M Description of Requirement',
+      '6N Purchase Card as Payment Method',
+      '6R National Interest Action',
+      '9A DUNS Number',
+      '9B Contractor Name from Contract',
+      '10J Commercial Item Test Program',
+      '10M Solicitation Procedures'
```

Comment:
CodeClimate does not love the size of this class. Maybe stick the above array into its own class or module called something like `WinningBidderExportFields`.